src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -209,7 +209,18 @@ MOBILE_INTEGRATION_TEST_TOKEN = '{{ MOBILE_INTEGRATION_TEST_TOKEN }}'
 
 ####### For Constructing Global URLs ########
 
-LOCAL_APPS = ('dimagi.utils', 'gunicorn',)
+LOCAL_APPS = (
+    'dimagi.utils',
+    'gunicorn',
+    {% for entry in LOCAL_APPS.apps %}
+    {% if inventory_hostname in groups[entry.host] %}
+    '{{ entry.name }}'
```

Comment:
I think you still need a comma at the end of this line. Otherwise it will be broken if there are multiple items in `LOCAL_APPS.apps`.

Reply:
Makes sense, https://github.com/dimagi/commcare-cloud/pull/2248/commits/692ac06cb82493d4ee9f2482fe9ed69e01d676cf